ports/bare-arm/mpconfigport.h

```diff
@@ -29,6 +29,7 @@
 #define MICROPY_PY_BUILTINS_SET     (0)
 #define MICROPY_PY_BUILTINS_SLICE   (0)
 #define MICROPY_PY_BUILTINS_PROPERTY (0)
+#define MICROPY_PY_BUILTINS_STR_OP_MODULO (0)
```

Comment:
IMO `MICROPY_BUILTINS_STR_MODULO` is descriptive enough, no need for the "OP" bit.

Reply:
Per conventions, MICROPY_BUILTINS_STR_MODULO would mean builtins.str.modulo(). I'm trying to disambiguate that, taking MP_BINARY_OP_MODULO as a reference. If you think it's not needed, I can drop "OP_".

Reply:
Yes there is ambiguity.  There already exists `MICROPY_PY_BUILTINS_RANGE_BINOP` to cover all binary ops (not `range.binop()`!), so maybe even `MICROPY_PY_BUILTINS_STR_BINOP_MODULO` is the consistent thing to do here.

Reply:
Well, it's called BINOP, because there's no other designator for "OP". "OP_MODULO" should be pretty disambiguating and not longer than necessary. So, please confirm the final choice.